docs/_templates/layout.html

```diff
@@ -0,0 +1,16 @@
+{% extends "!layout.html" %}
+{% block extrahead %}
+<script type="text/javascript">
+  // Configure TMS settings
+  var wapLocalCode = 'us-en'; // Dynamically set per localized site; see mapping table for values
+  var wapSection = "oneapi-mkl"; // WAP team will give you a unique section for your site
+  // Load TMS
+  (function () {
+    var url = 'https://www.intel.com/content/dam/www/global/wap/tms-loader.js'; // WAP file URL
+    var po = document.createElement('script'); po.type = 'text/javascript'; po.async = true; po.src = url;
+    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(po, s);
+  })();
+</script>
+{% endblock %}
+<!-- any other content blocks -->
+<!--...-->
```

Comment:
Should this html page be included to any other pages?

Reply:
Good question: The Sphinx build takes care of that (as specified by variables in the config.py file).